### DISCUSSION TITLE
當機器手臂出現在開刀現場，背後的資安如何加強

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#696] [General]
我曾經在美劇裡看過機械手臂的開刀場景，現在台灣的醫療技術竟然也使用了機械手臂！

我在想若機械手臂有連網的話，那肯定是有資安的問題的。趨勢科技全球部門指出，全世界的醫療駭客還是很多的，例如：2019年美國一間醫療院所曾發生勒索攻擊事件，因為重要系統被攻擊而造成無法讀取病患資料、醫護人員無法隨時監督重要的心率資訊等影響。
若沒有連網的話還是有被駭客入侵的可能性，雖然較低，但還是不可忽略，如插入帶有病毒的隨身碟、或是無限電波（藍芽）的干擾之類的。

機械手臂的資安維護在文章中汪映廷建議：「生產手術機器人的設備商最好要經過受到認可的資安認證，證明本身產品是沒有問題的，那就已經降低大部分的資安風險、屬於操作端的醫療場域，則最好偕同資安公司進行相關評估，如資安架構檢視、滲透測試等等，來降低人為設定錯誤」

因此我認為防毒軟體在醫療科技產品上仍有很大的進步空間！

文章連結：https://www.thenewslens.com/article/176382/fullpage

**Comment by Vincent20030314** [comment]
這項技術能使手術成功機率更大，但是相對的必須做好防毒準備，否則一條命可能就會瞬間消失在眼前。
